ddtrace/_trace/span.py

```diff
@@ -813,13 +813,13 @@ def __repr__(self) -> str:
             self.name,
         )
 
+    @property
+    def is_top_level(self) -> bool:
```

Comment:
Making this private with a leading underscore would avoid adding unnecessary surface area to the public API.